packages/cli/src/ui/hooks/slashCommandProcessor.ts

```
@@ -274,581 +272,229 @@ export const useSlashCommandProcessor = (
-              return;
-            }
-            case 'resume':
-            case 'restore':
-            case 'load': {
+            case 'save':
+              await logger.saveCheckpoint(chat.getHistory() || [], tag);
+              addMessage({ type: MessageType.INFO, content: `Conversation checkpoint saved${tag ? ' with tag: ' + tag : ''}.`, timestamp: new Date() });
+              break;
+            case 'resume': case 'restore': case 'load':
               const conversation = await logger.loadCheckpoint(tag);
               if (conversation.length === 0) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)